code/controllers/subsystems/job.dm

```diff
@@ -745,6 +745,9 @@
 				to_chat(H, "<span class='warning'>Your current job or whitelist status does not permit you to spawn with [thing]!</span>")
 				continue
 
+			if(G.augment) //augments are handled somewhere else
+				continue
```

Comment:
Most of this proc is the same as the new one. Isn't it possible to just do the organ stuff in this if and then return?

Reply:
Can't be in the same proc, because when you join in the arrivals you don't get your clothing, but you surely should get your organs.